package.json

```diff
@@ -95,7 +91,7 @@
     "dn.publish": "npm run dn.build && dotnet publish -c release openapi-diff/src/core/OpenApiDiff/OpenApiDiff.csproj",
     "dn.test": "npm run dn.build && dotnet test openapi-diff/src/modeler/AutoRest.Swagger.Tests/AutoRest.Swagger.Tests.csproj",
     "tsc": "tsc",
-    "ts.test": "tsc && jest",
+    "ts.test": "jest src/**",
```

Comment:
Why remove `tsc` and why add `src/**` ?

Reply:
https://github.com/Azure/openapi-diff/pull/340#discussion_r1651815699

This comment explains the reasoning for removing the `tsc`, but the src/** is indeed unecessary, I'll remove it!